testsuite/forge/src/backend/k8s/cluster_helper.rs

```diff
@@ -236,18 +260,39 @@ async fn delete_k8s_collection<T: Clone + DeserializeOwned + Meta>(
 pub(crate) async fn delete_k8s_resources(client: K8sClient, kube_namespace: &str) -> Result<()> {
     // selector for the helm chart
     let aptos_node_helm_selector = "app.kubernetes.io/part-of=aptos-node";
+    let testnet_addons_helm_selector = "app.kubernetes.io/part-of=testnet-addons";
+    let genesis_helm_selector = "app.kubernetes.io/part-of=aptos-genesis";
 
     // delete all deployments and statefulsets
     // cross this with all the compute resources created by aptos-node helm chart
     let deployments: Api<Deployment> = Api::namespaced(client.clone(), kube_namespace);
     let stateful_sets: Api<StatefulSet> = Api::namespaced(client.clone(), kube_namespace);
     let pvcs: Api<PersistentVolumeClaim> = Api::namespaced(client.clone(), kube_namespace);
-    delete_k8s_collection(deployments, "deployments", aptos_node_helm_selector).await?;
-    delete_k8s_collection(stateful_sets, "stateful_sets", aptos_node_helm_selector).await?;
-    delete_k8s_collection(pvcs, "pvcs", aptos_node_helm_selector).await?;
+    let jobs: Api<Job> = Api::namespaced(client.clone(), kube_namespace);
+    let cronjobs: Api<CronJob> = Api::namespaced(client.clone(), kube_namespace);
+    // service deletion by label selector is not supported in this version of k8s api
+    // let services: Api<Service> = Api::namespaced(client.clone(), kube_namespace);
```

Comment:
nit: do we want to leave these lines uncommented here? Not a biggie, but just want to make sure it's indended.

Reply:
Yea it's intended and the above comment describes why it's commented out :)